{"pytest_version": "7.4.3", "$report_type": "SessionStart"}
{"nodeid": "", "outcome": "passed", "longrepr": null, "result": null, "sections": [], "$report_type": "CollectReport"}
{"nodeid": "tests27_88_4.py", "outcome": "passed", "longrepr": null, "result": null, "sections": [], "$report_type": "CollectReport"}
{"nodeid": "tests27_88_4.py::test_presence_of_inserted_element", "location": ["tests27_88_4.py", 4, "test_presence_of_inserted_element"], "keywords": {"test_presence_of_inserted_element": 1, "tests27_88_4.py": 1, "Folder_88": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00013663596473634243, "start": 1701648826.8322105, "stop": 1701648826.8323476, "$report_type": "TestReport"}
{"nodeid": "tests27_88_4.py::test_presence_of_inserted_element", "location": ["tests27_88_4.py", 4, "test_presence_of_inserted_element"], "keywords": {"test_presence_of_inserted_element": 1, "tests27_88_4.py": 1, "Folder_88": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/Benchmark_GPT3_5/Q27/gpt_3_5_turbo_results_4/Folder_88/generated_answer.py", "lineno": 3, "message": "ValueError: 91 is not in list"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_presence_of_inserted_element():", "        initial_list = [set() for i in range((91 + 1) * 2)]", ">       returned_list = insert_after_index(initial_list)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q27/gpt_3_5_turbo_results_4/Folder_88/tests27_88_4.py", "lineno": 7, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def insert_after_index(lst):", ">       index = lst.index(91) + 1", "E       ValueError: 91 is not in list"], "reprfuncargs": {"args": [["lst", "[set(), set(), set(), set(), set(), set(), ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q27/gpt_3_5_turbo_results_4/Folder_88/generated_answer.py", "lineno": 3, "message": "ValueError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_presence_of_inserted_element():", "        initial_list = [set() for i in range((91 + 1) * 2)]", ">       returned_list = insert_after_index(initial_list)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q27/gpt_3_5_turbo_results_4/Folder_88/tests27_88_4.py", "lineno": 7, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def insert_after_index(lst):", ">       index = lst.index(91) + 1", "E       ValueError: 91 is not in list"], "reprfuncargs": {"args": [["lst", "[set(), set(), set(), set(), set(), set(), ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q27/gpt_3_5_turbo_results_4/Folder_88/generated_answer.py", "lineno": 3, "message": "ValueError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/Benchmark_GPT3_5/Q27/gpt_3_5_turbo_results_4/Folder_88/generated_answer.py", "lineno": 3, "message": "ValueError: 91 is not in list"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00013728707563132048, "start": 1701648826.832471, "stop": 1701648826.8326087, "extras": [], "$report_type": "TestReport"}
{"nodeid": "tests27_88_4.py::test_presence_of_inserted_element", "location": ["tests27_88_4.py", 4, "test_presence_of_inserted_element"], "keywords": {"test_presence_of_inserted_element": 1, "tests27_88_4.py": 1, "Folder_88": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [], "duration": 0.00012200709898024797, "start": 1701648826.8379598, "stop": 1701648826.8380828, "$report_type": "TestReport"}
{"exitstatus": 1, "$report_type": "SessionFinish"}
